lib/cosmos/packets/commands.rb

```diff
@@ -261,6 +261,28 @@ def clear_counters
       end
     end
 
+    # Returns an array with a "TARGET_NAME PACKET_NAME" string for every command in the system (PACKET_NAME == command name)
+    def all_item_strings(include_hidden = false, splash = nil)
```

Comment:
Please rename this to all_packet_strings.  Otherwise looks good. Thanks!

Reply:
Done!